apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -199,6 +208,44 @@ private void setSenderId() {
         Push.setSenderId(SENDER_ID);
     }
 
+    private void setMaxStorageSize() {
+        final long maxStorageSize = sSharedPreferences.getLong(MAX_STORAGE_SIZE_KEY, 0);
+        if (maxStorageSize <= 0) {
+            return;
+        }
+        // TODO remove reflection once new APIs available in jCenter.
```

Comment:
You don't need to address this comment since we will remove this line quite soon but make sure you have a new line before any comments.